### PR TITLE
refactor: use self-closing tags 

### PR DESCRIPTION
**Pull request by dzhavat**
### Workspace
- configure self-closing tags lint rule to only target files that end with `*.component.html`. This also includes inline templates.
- run lint on the workspace
- replace deprecated `dep-graph` Nx command

Note: main `index.html` files should not use self-closing custom elements (i.e. `<dh-app />`), because, as per HTML spec, this is invalid HTML.

## References

- N/A

### CODE REVIEW

**Review by mimse — APPROVED** [review]

File: .eslintrc.json [hunk 1]
```diff
@@ -205,6 +205,13 @@
       "files": ["*.ts"],
       "extends": ["plugin:@ngrx/component-store-strict"]
     },
+    {
+      "files": ["apps/**/app/**/*.html", "libs/**/*.html"],
```
Comment (frontend-specialisten) [inline .eslintrc.json:209]: Question: what about components using inline templates?

**Review by frontend-specialisten — COMMENTED** [review]
1. EO has now been checked for failures, and everything now works as expected.  
2. When I try breaking the rule, I don't get any feedback in my editor or by running `yarn lint`?
3. Can you please explain this: `Note: main index.html files should not use self-closing custom elements (i.e. <dh-app />) as this is invalid HTML.`

File: .eslintrc.json (hunk 1 above)
Comment (dzhavat) [reply .eslintrc.json:209]: It looks like the parser used in the next line also checks inline templates. :)

Comment (dzhavat) [reply .eslintrc.json]: I tested a few things, and the `parser` was not really needed. The inline templates are still checked, though.

**Review by frontend-specialisten — APPROVED** [review]
Great work!

**Pull request merged**